js/admin/instructorNew.js

```diff
@@ -0,0 +1,221 @@
+// // instatiation of all mdc components
+// // text fields
+const textFields = document.querySelectorAll('.mdc-text-field');
+for (const textField of textFields) {
+  mdc.textField.MDCTextField.attachTo(textField);
+}
+
+// cards
+const selector = '.mdc-card__primary-action, .mdc-fab .mdc-button';
+
+const ripples = [].forEach.call(document.querySelectorAll(selector), el => {
+    mdc.ripple.MDCRipple.attachTo(el);
+});
+
+
+// dialogs
+const dialogs = document.querySelectorAll('.mdc-dialog');
+dialogs.forEach(dialog => {
+ const modal = new mdc.dialog.MDCDialog(dialog);
+})
```

Comment:
Store a reference to each dialog you instantiate. 
This allows you to use the methods on the MDCDialog instance, such as `open` to open the modal, and `close` to close the modal.

Perhaps you could store them in a object, using a `data-modal-id` attribute or similar as a key for the object.
So:
```javascript
const dialogs = document.querySelectorAll('.mdc-dialog');
const dialogInstances = {}
dialogs.forEach(dialog => {
  const modalId = dialog.dataset.modalId;
  dialogInstances[modalId] = new mdc.dialog.MDCDialog(dialog);
})
```

Reply:
done